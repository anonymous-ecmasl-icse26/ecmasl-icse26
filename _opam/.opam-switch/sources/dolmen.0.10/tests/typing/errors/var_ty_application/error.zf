
goal exists (a : type). a a.

