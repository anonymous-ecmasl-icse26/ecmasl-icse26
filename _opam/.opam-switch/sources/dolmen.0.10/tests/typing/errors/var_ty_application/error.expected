File "tests/typing/errors/var_ty_application/error.zf", line 2, character 24-27:
2 | goal exists (a : type). a a.
                            ^^^
Error Cannot apply arguments to type variable a
